### ISSUE TITLE
Use raven.js

### ISSUE THREAD

**Issue by jmartinm** [inspirehep/inspire-next#2382]
https://docs.sentry.io/clients/javascript/

We need to check the specifics of using it with AngularJS (for Holding Pen) and with Angular 2+ for the editor.

**Comment by jmartinm** [comment]
@harunurhan You can start looking into this

**Comment by harunurhan** [comment]
@jmartinm for holding pen, there is simple way but it's not `requirejs` way. 

after angular is loaded this should be executed.
```html
<script src="/node_modules/raven-js/dist/raven.js"></script>
<script src="/node_modules/raven-js/dist/plugins/angular.js"></script>
<script>
  Raven
    .config('https://<key>@sentry.io/<project>')
    .addPlugin(Raven.Plugins.Angular)
    .install();
</script>
```

Although there are some examples, but none for requirejs. However if this is not good enough I can look into it.

**Comment by jmartinm** [comment]
Blocked by the version of our Sentry server (6.4.4)

From ravenjs:

```
For on-premise users:

Raven.js 3.6 requires Sentry server 8.6.0 or later
Raven.js 2.0 requires Sentry server 8.0.0 or later
```

**Comment by jmartinm** [comment]
@harunurhan can we close this?

**Comment by harunurhan** [comment]
Actually I haven't set it up for inspire-next yet, only for other angular projects.

**Comment by jmartinm** [comment]
OK, it would be nice to have in submission forms and Holding Pen at least, which are both used in production. Submission forms don't use Angular so no need to add the plugin.

**Comment by harunurhan** [comment]
ok  I can set it up?, Shouldn't be difficult, if we don't need to do sourcemap uploads
